packages/angular-cli/tasks/build-webpack.ts

```diff
@@ -2,41 +2,22 @@ import * as rimraf from 'rimraf';
 import * as path from 'path';
 const Task = require('../ember-cli/lib/models/task');
 import * as webpack from 'webpack';
-import { BuildOptions } from '../commands/build';
+import { BuildTaskOptions } from '../commands/build';
 import { NgCliWebpackConfig } from '../models/webpack-config';
-import { getWebpackStatsConfig } from '../models/';
+import { getWebpackStatsConfig } from '../models/webpack-configs/utils';
 import { CliConfig } from '../models/config';
 
 
 export default Task.extend({
-  run: function (runTaskOptions: BuildOptions) {
+  run: function (runTaskOptions: BuildTaskOptions) {
 
     const project = this.cliProject;
 
-    const outputDir = runTaskOptions.outputPath || CliConfig.fromProject().config.apps[0].outDir;
-    const deployUrl = runTaskOptions.deployUrl ||
```

Comment:
I refactored the tests around a bit and that test is in `tests/e2e/tests/build/styles/extract-css.ts`, the last test. Can you double check if that's right?